pkg/system/stat_windows.go

```diff
@@ -3,15 +3,34 @@
 package system
 
 import (
-	"errors"
-	"syscall"
+	"os"
+	"time"
 )
 
-func fromStatT(s *syscall.Win32FileAttributeData) (*Stat_t, error) {
-	return nil, errors.New("fromStatT should not be called on windows path")
+type Stat_t struct {
+	name    string
+	size    int64
+	mode    os.FileMode
+	modTime time.Time
+	isDir   bool
 }
 
-func Stat(path string) (*Stat_t, error) {
-	// should not be called on cli code path
```

Comment:
awesome you are implementing all that :)  maybe `ErrNotSupportedPlatform` is obsolete now?